### DISCUSSION TITLE
Cannot Read Aloud... Can Only Download the Audio File!

### DISCUSSION THREAD

**Discussion by seanmodd** [pgmichael/wavenet-for-chrome#74] [Q&A]
I have a very annoying problem... please help! My extension will not read aloud anything, I can only hear the TTS once I have downloaded the audio file!

For example:
1. If I right click > wavenet for chrome > read aloud it doesn't work.
2. If I use the shortcut, read aloud doesn't work.
3. If I go to the extension's sandbox and enter a sample text, read aloud doesn't work.

In all three instances however, I can download the audio file and it works perfectly.
I have also uninstalled and reinstalled the extension and still have the same problem.
I have also created a new API key and still have the same problem. 

I'm not sure what else I can do but it's not working... please help!

Thank you!

**Comment by pgmichael** [comment]
Hi sean,

What browser, browser version and operating system are you using? Does this seem to occur on every website or only specific ones?
